CorsixTH/Lua/game_ui.lua

```diff
@@ -541,6 +541,13 @@ function GameUI:onMouseMove(x, y, dx, dy)
     return false
   end
 
+  local mouse_drag_button = nil
+  if self.app.config.right_mouse_panning then
+    mouse_drag_button = self.buttons_down.mouse_right
+  else
+    mouse_drag_button = self.buttons_down.mouse_middle
+  end
+
```

Comment:
Thanks for the tip, but this will cause it to always return true when either the middle or right mouse button is pressed, when right_mouse_panning is set to true. However, it will never return true when right_mouse_panning is not set.

Reply:
That's not true, and is higher prescedent.
e and r or m

```
erm =
000 0
100 0
010 0
110 1
001 1
...
```

Reply:
Not strictly true what you just said but not accurate either if we don't want middle to pan.